.changeset/tough-candles-travel.md

```diff
@@ -0,0 +1,153 @@
+---
+'@makeswift/controls': patch
+'@makeswift/runtime': patch
+---
+
+### New `Font` control
+
+We now have a `Font` control. This control let's you select a `fontFamily`, `fontStyle`, and `fontWeight`.
+The values available are sourced from our Google Fonts integration within Makeswift and from the variants you pass to `getFonts` in your `[...makeswift]` route handler.
```

Comment:
Let's change `[...makeswift]` to `MakeswiftApiHandler`. Also, maybe we can link to the docs?